src/ewatercycle/plugins/hype/model.py

```diff
@@ -137,29 +136,33 @@ def _export_config(self) -> Path:
 
         return cfg_file
 
-    def _make_cfg_dir(self, cfg_dir: Optional[Path] = None) -> Path:
-        if cfg_dir:
-            cfg_dir = to_absolute_path(cfg_dir)
-        else:
-            # Must exist before setting up default config
-            timestamp = datetime.datetime.now(datetime.timezone.utc).strftime(
-                "%Y%m%d_%H%M%S"
-            )
-            cfg_dir = to_absolute_path(f"hype_{timestamp}", parent=CFG.output_dir)
-        cfg_dir.mkdir(parents=True, exist_ok=True)
+    def _make_cfg_dir(
+        self,
+        cfg_dir: Optional[str] = None,
+        **kwargs,
+    ) -> Path:
+        """Make sure there is a working directory.
+
+        Args:
+            cfg_dir: If cfg dir is None or does not exist then create sub-directory
+                in CFG.output_dir
+        """
+        cfg_path = super()._make_cfg_dir(
+            cfg_dir=cfg_dir, folder_prefix="hype", **kwargs
```

Comment:
Using `self.__class__.__name__.lower()` would probably be a nice way of retrieving the folder prefix. It would usually remove the need to define `_make_cfg_dir`